source/client/options_impl.cc

```diff
@@ -140,8 +140,8 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
 
   TCLAP::ValueArg<uint32_t> max_active_requests(
       "", "max-active-requests",
-      fmt::format("Max active requests (default: {}).", max_active_requests_), false, 0, "uint32_t",
-      cmd);
+      fmt::format("Max active requests (default: {}). HTTP/2 only.", max_active_requests_), false,
+      0, "uint32_t", cmd);
```

Comment:
This seems to say that the default is zero.

Reply:
We ignore TCLAP's capability to set defaults. Instead, we define the (trivial) defaults in the option_impl header. Also added a comment above to point that out to future readers.